documentation/staging/layouts/shortcodes/latestVersion.html

```diff
@@ -1 +1 @@
-4.0.0
+4.0.0
```

Comment:
Just a little note on this change: I removed the new-line because if the file ends in a new-line, then the short-code in-lines a new-line into markup that uses the short-code, and that's a problem...